jobs/build/appregistry/Jenkinsfile

```diff
@@ -132,25 +149,35 @@ node {
                     return
                 }
 
-                currentBuild.description += "\npushing operator metadata."
-                withCredentials([usernamePassword(credentialsId: 'quay_appregistry_omps_bot', usernameVariable: 'QUAY_USERNAME', passwordVariable: 'QUAY_PASSWORD')]) {
-                    def token = retrieveBotToken()
-                    for (def i = 0; i < operatorData.size(); i++) {
-                        def build = operatorData[i]
-                        retry(3) {
-                            def response = httpRequest(
-                                url: "https://omps-prod.cloud.paas.psi.redhat.com/v2/redhat-operators-art/koji/${build.nvr}",
-                                httpMode: 'POST',
-                                customHeaders: [[name: 'Authorization', value: token]],
-                                timeout: 60,
-                                validResponseCodes: "200:599",
-                            )
-                            if (response.status != 200) {
-                                sleep(5)
-                                error "OMPS request for ${build.nvr} failed: ${response.status} ${response.content}"
+                if (params.STREAM == 'dev') {
+                    currentBuild.description += "\npushing operator metadata."
+                    withCredentials([usernamePassword(credentialsId: 'quay_appregistry_omps_bot', usernameVariable: 'QUAY_USERNAME', passwordVariable: 'QUAY_PASSWORD')]) {
+                        def token = retrieveBotToken()
+                        for (def i = 0; i < operatorData.size(); i++) {
+                            def build = operatorData[i]
+                            retry(3) {
+                                def response = httpRequest(
+                                    url: "https://omps-prod.cloud.paas.upshift.redhat.com/v2/redhat-operators-art/koji/${build.metadata_nvr}",
+                                    httpMode: 'POST',
+                                    customHeaders: [[name: 'Authorization', value: token]],
+                                    timeout: 60,
+                                    validResponseCodes: "200:599",
+                                )
+                                if (response.status != 200) {
+                                    sleep(5)
+                                    error "OMPS request for ${build.metadata_nvr} failed: ${response.status} ${response.content}"
+                                }
                             }
+                            currentBuild.description += "\n  ${build.metadata_nvr}"
                         }
-                        currentBuild.description += "\n  ${build.nvr}"
+                    }
+                } else {
+                    if (params.ADVISORY) {
+                        cmd = "--group openshift-${params.BUILD_VERSION} "
```

Comment:
You could simplify this to something like (i didn't type everything out)

```
cmd = """--group openshift-${params}....
find-builds -k image
--build ${build...}
--attach ${params...}"""
buildlib.elliott(cmd)
```

Reply:
sure!

Reply:
cool. with or without that change I give this a solid LGTM